### PR TITLE
Integrate API with example

### PR DESCRIPTION
**Pull request by kien-to**
1. Connected the backend with the frontend using Axios in the "**services**" folder. Insider the "**services**" folder there are 2 files:
- apiService containing the localhost link and axios connection. You don't have to modify this.
- apiSection containing the function 'sectionDetails' that makes request to the backend api to get section information.
2. Add an example of fetching sections from the backend in /pages/index.tsx (Using React-Query).
- sections is returning all sections data from the backend, which you can use to display to the frontend.
- console.log(sections) will show those data.

### CODE REVIEW

**Review by dhrubabasu — CHANGES_REQUESTED** [review]

File: yarn.lock
[the remark this answers is not in the document]
Comment (kien-to) [reply yarn.lock:1]: Okay

[the remark this answers is not in the document]
Comment (tinholeong020) [reply yarn.lock:1]: Why I got to review lol, making me do stuff
smh

**Review by tinholeong020 — APPROVED** [review]
LGTM

**Comment by TCarel01** [comment]
LGTM

**Pull request merged**